src/ui/common/src/components/workflows/edges/AqueductQuadratic.tsx

```diff
@@ -25,7 +25,6 @@ const AqueductQuadratic: React.FC<EdgeProps> = ({
       <defs>
         <marker
           id="arrow-closed"
-          viewBox="-10 -10 20 20"
```

Comment:
Instead of removing this can you comment it out and leave a TODO comment to investigate later? I'm curious about this as I'm not getting this linter error on my end and want to investigate further later.